alembic/versions/0011.py

```diff
@@ -0,0 +1,119 @@
+VALUES(1, 1, 1, '{"1": "Паспорт громадянина України (оригінал)",
+                "2": "6 фотокарток 3х4",
+                "3": "Результат флюрографії, з терміном видач до одного року (оригінал)",
+                "4": "Квитанція про сплату за проживання не менше ніж за 4 місяці (копія)"}'::JSON);
+
+INSERT INTO speciality(faculty_id, code, name) VALUES (1, 051, 'Економіка');
+INSERT INTO speciality(faculty_id, code, name) VALUES (1, 121, 'Інженерія програмного забезпечення');
+INSERT INTO speciality(faculty_id, code, name) VALUES (1, 122, 'Комп''ютерні науки');
+INSERT INTO speciality(faculty_id, code, name) VALUES (1, 124, 'Системний аналіз');
+INSERT INTO speciality(faculty_id, code, name) VALUES (1, 125, 'Кібербезпека');
+INSERT INTO speciality(faculty_id, code, name) VALUES (1, 126, 'Інформаційні системи та технології');
+INSERT INTO speciality(faculty_id, code, name) VALUES (1, 186, 'Видавництво та поліграфія');
+INSERT INTO speciality(faculty_id, code, name) VALUES (6, 051, 'Економіка');
+INSERT INTO speciality(faculty_id, code, name) VALUES (6, 053, 'Психологія');
+INSERT INTO speciality(faculty_id, code, name) VALUES (6, 081, 'Право, освітня програма');
+INSERT INTO speciality(faculty_id, code, name) VALUES (6, 232, 'Соціальне забезпечення');
+INSERT INTO speciality(faculty_id, code, name) VALUES (6, 281, 'Публічне управління та адміністрування');
+INSERT INTO speciality(faculty_id, code, name) VALUES (5, 073, 'Менеджмент');
+INSERT INTO speciality(faculty_id, code, name) VALUES (5, 075, 'Маркетинг');
+INSERT INTO speciality(faculty_id, code, name) VALUES (5, 022, 'Дизайн');
+INSERT INTO speciality(faculty_id, code, name) VALUES (4, 072, 'Фінанси, банківська справа та страхування');
+INSERT INTO speciality(faculty_id, code, name) VALUES (4, 071, 'Облік і оподаткування');
+INSERT INTO speciality(faculty_id, code, name) VALUES (2, 011, 'Освітні, педагогічні науки');
+INSERT INTO speciality(faculty_id, code, name) VALUES (2, 052, 'Політологія');
+INSERT INTO speciality(faculty_id, code, name) VALUES (2, 061, 'Журналістика');
+INSERT INTO speciality(faculty_id, code, name) VALUES (2, 073, 'Менеджмент');
+INSERT INTO speciality(faculty_id, code, name) VALUES (2, 291, 'Міжнародні відносини, суспільні комунікації та регіональні студії');
+INSERT INTO speciality(faculty_id, code, name) VALUES (2, 292, 'Міжнародні економічні відносини');
+INSERT INTO speciality(faculty_id, code, name) VALUES (3, 051, 'Економіка');
+INSERT INTO speciality(faculty_id, code, name) VALUES (3, 073, 'Менеджмент');
+INSERT INTO speciality(faculty_id, code, name) VALUES (3, 076, 'Підприємництво, торгівля та біржова діяльність');
+INSERT INTO speciality(faculty_id, code, name) VALUES (3, 241, 'Готельно-ресторанна справа');
+INSERT INTO speciality(faculty_id, code, name) VALUES (3, 242, 'Туризм');
+
+INSERT INTO student(last_name, first_name, middle_name, telephone_number, gender, course_id, speciality_id, faculty_id)
```

Comment:
That's for saving time. I think we'll create a hundred and more databases for personal testing, and it may be a good idea to have student account ready for usage. Of course we can delete it but we can a real admin account for testing purpose

Reply:
Deleted test data